Example/URBNDataSource/ViewControllers/Code CollectionView/URBNSwiftAccordionTableViewController.swift

```diff
@@ -0,0 +1,155 @@
+//
+//  URBNSwiftAccordionTableViewController.swift
+//  URBNDataSource
+//
+//  Created by Dustin Bergman on 12/15/15.
+//  Copyright © 2015 Joe. All rights reserved.
+//
+
+import UIKit
+
+typealias CellTappedBlock = () -> Void
+
+class URBNSwiftAccordionTableViewController: UITableViewController {
+
+    lazy var adapter: URBNAccordionDataSourceAdapter = {
+        var items = [[String]]()
+        var sections = [String]()
+        for i in 0...5 {
+            sections.append("Section " + String(i))
+            items.append(["Item 0", "Item 1", "Item 2", "Item 3", "Item 4"])
+        }
+        
+        return URBNAccordionDataSourceAdapter(sectionObjects: sections, andItems: items)
+    }()
+
+    @IBOutlet var stepper: UIStepper!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        var items = [[String]]()
+        var sections = [String]()
+        for i in 0...5 {
+            sections.append("Section " + String(i))
+            items.append(["Item 0", "Item 1", "Item 2", "Item 3", "Item 4"])
+        }
+        
+        stepper.value = Double(sections.count)
+
+        adapter.fallbackDataSource = self
+        adapter.tableView = tableView
+        adapter.allowMultipleExpandedSections = true
+        
+        /// If all of your cell classes are unique, then you can just call regsiter cell with that class.
+        /// The identifier will be the className
+        adapter.registerCellClass(UITableViewCell.self) { (cell, object, indexPath) in
+            guard let cell = cell as? UITableViewCell,
+            let object = object as? String else {
```

Comment:
this part of data source currently drives me up the wall. the swift data source is under construction hopefully this will be an obsolete use case soon enough